content/actions/using-workflows/reusing-workflows.md

```diff
@@ -265,6 +265,7 @@ When you call a reusable workflow, you can only use the following keywords in th
 
   - If `jobs.<job_id>.permissions` is not specified in the calling job, the called workflow will have the default permissions for the `GITHUB_TOKEN`. For more information, see "[AUTOTITLE](/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)."
   - The `GITHUB_TOKEN` permissions passed from the caller workflow can be only downgraded (not elevated) by the called workflow.
+  - Avoid using `jobs.<job_id>.concurrency.cancel-in-progress: true` and same `jobs.<job_id>.concurrency.group` values between called and caller workflows to prevent immediate cancel on called workflow
```

Comment:
@mahdiar-naufal-shyftplan - Thank you for creating this pull request. I think this is a useful clarification here. However, I found this difficult to follow until I went and read the related community discussion, so I feel like this needs expanded a little to provide more of an explanation. I suggest changing this to:

```suggestion
  - If you use `jobs.<job_id>.concurrency.cancel-in-progress: true`, don't use the same value for `jobs.<job_id>.concurrency.group` in the called and caller workflows as this will cause the workflow that's already running to be cancelled. A called workflow uses the name of its caller workflow in {% raw %}${{ github.workflow }}{% endraw %}, so using this context as the value of `jobs.<job_id>.concurrency.group` in both caller and called workflows will cause the caller workflow to be cancelled when the called workflow runs. 
```